res/openvpn-gui-res-fr.rc

```diff
@@ -75,12 +75,12 @@ END
 /* Challenge Response Dialog */
 ID_DLG_CHALLENGE_RESPONSE DIALOG 6, 18, 212, 72
 STYLE WS_SIZEBOX | WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
-CAPTION "OpenVPN - Challenge Response"
+CAPTION "OpenVPN"
```

Comment:
Any particular reason to omit translating "Challenge Response" in this dialog?

Reply:
C'est une erreur de ma part oui cette ligne ne devrait pas etre traduite
Merci

Reply:
@bubuleur : if you revert this particular change then I think this PR is good to go. My wife took a quick look at the translations and they looked good.